bower.json

```diff
@@ -19,18 +19,19 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-tuples":        "~0.3.0",
-    "purescript-either":        "~0.1.4",
-    "purescript-monoid":        "~0.2.0",
-    "purescript-exceptions":    "~0.2.2",
-    "purescript-control":       "~0.2.2",
-    "purescript-maybe":         "~0.2.1",
-    "purescript-strings":       "~0.4.5",
-    "purescript-monad-eff":     "~0.1.0",
-    "purescript-lists":         "~0.6.0",
-    "purescript-profunctor":    "~0.2.1",
-    "purescript-control":       "~0.2.6",
-    "purescript-foldable-traversable": "~0.3.1",
-    "purescript-transformers":  "~0.5.1"
+    "purescript-tuples":                "^0.4.0",
+    "purescript-either":                "^0.2.0",
+    "purescript-monoid":                "^0.3.0",
+    "purescript-exceptions":            "^0.3.0",
+    "purescript-control":               "^0.3.0",
+    "purescript-maybe":                 "^0.3.0",
+    "purescript-strings":               "^0.5.0",
+    "purescript-eff":                   "^0.1.0",
+    "purescript-lists":                 "^0.7.0",
+    "purescript-profunctor":            "^0.3.0",
+    "purescript-control":               "^0.3.0",
+    "purescript-console":               "^0.1.0",
+    "purescript-foldable-traversable":  "^0.4.0",
+    "purescript-transformers":          "^0.6.0"
   }
```

Comment:
This is just being picky, but can you reduce this to:
``` json
  "dependencies": {
    "purescript-console":      "^0.1.0",
    "purescript-exceptions":   "^0.3.0",
    "purescript-lists":        "^0.7.0",
    "purescript-profunctor":   "^0.3.0",
    "purescript-strings":      "^0.5.2",
    "purescript-transformers": "^0.6.1"
  }
```
That will pull everything in the above, and doesn't duplicate the `purescript-control` requirement :wink: